applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,118 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Desktop Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+The module chooses the highest frequency required by the application states and selects voltage accordingly.
+DVFS is critical for balancing performance and power consumption, especially in battery-operated devices.
+
+In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when the application performs less computation-challenging activities such as Bluetooth LE advertising.
+However, if performing more computation-consuming tasks nRF Desktop DVFS module will dynamically change to a higher frequency to increase performance.
+
+The nRF Desktop DVFS module requests clock frequency change using API of the clock control driver.
+The driver is responsible for choosing the optimal clock frequency.
+The driver submits a request to the sysctrl core through the nrfs DVFS service.
+The sysctrl core controls both clock frequency and voltage and chooses corresponding voltage to requested frequency.
+
+.. note::
+   In the current solution, it is assumed that no other module requests frequency changes.
+   The nRF Desktop DVFS module is the sole entity responsible for managing frequency and voltage adjustments.
+   Controlling clock from other sources would result in a module fatal error.
+
+Supported devices
+*****************
+
+The module is only available for nRF54H20 SoC.
+
+The nRF54H20 SoC has the following available operating points (in documentation called frequency):
+
+1. DVFS_FREQ_HIGH - high frequency - 320 MHz, 0.8 V
+#. DVFS_FREQ_MEDLOW - medium-low frequency - 128 MHz, 0.6V
+#. DVFS_FREQ_LOW - low frequency - 64 MHz, 0.5V
```

Comment:
We do not use the DVFS API directly anymore, maybe we should just specify it as HIGH/MEDLOW/LOW as the enum does not matter here (it is used under the hood)?

Reply:
It matters in a way that you cannot have different values. I would leave it as it is to show where from those values came from.